index.html

```diff
@@ -534,7 +538,22 @@ <h4>Ravi Saraswat</h4>
         </div>
       </div>
 
-
+	<div class="subcontent" id="projects">
+        <nav class="navbar navbar-expand-lg navbar-light bg-light">
+	  <div class="container-fluid">
+	    <button type="button" class="btn btn-info sidebarCollapse">
+	      <i class="fas fa-align-left"></i>
+	    </button>
+	  </div>
+	</nav>	
+	<div class="container projects">
+	  <div class="container1">
+	    <div class="main">
+	      <div class=heading> PROJECTS ON CODESHOWS </div>
+	    </div>
```

Comment:
Add the existing repositories as projects with description and link from the readme